precompiles/avs/IAVSManager.sol

```diff
@@ -96,6 +99,7 @@ interface IAVSManager {
         address slashAddr,
         address rewardAddr,
         string[] memory avsOwnerAddress,
+        string[] memory whitelistAddress,
```

Comment:
Why choose `string` type for these addresses and not `address` which is Solidity native and has great support? Each `exo1...` bech32-encoded address can be represented as a hex address and vice-versa.

Reply:
To keep the format uniform, the address list is an exo

Reply:
My question is more for each type where the `string address` is used; not just the newly added whitelist address list.